code/modules/paperwork/contract.dm

```diff
@@ -34,7 +34,7 @@
 /obj/item/paper/contract/employment/attack(mob/living/M, mob/living/carbon/human/user)
 	var/deconvert = FALSE
 	if(M.mind == target && !M.owns_soul())
-		if(user.mind && (user.mind.assigned_role == "Lawyer"))
+		if(user.mind && (user.mind.assigned_role == "Lawyer" || user.mind.assigned_role == "Corporate Affairs Agent"))
```

Comment:
```suggestion
		if(user.mind && user.mind.assigned_role == "Corporate Affairs Agent"))
```

Nobody will ever have their assigned role be lawyer if you're removing it as a roundstart job.